integration_tests/telemetry_async.cpp

```diff
@@ -232,3 +237,15 @@ void print_rc_status(Telemetry::RCStatus rc_status)
               << std::endl;
     _received_rc_status = true;
 }
+
+void print_position_velocity_local(Telemetry::PositionVelocityLocal position_velocity_local)
+{
+    std::cout << "Got position north_m: " << position_velocity_local.north_m << " m, "
```

Comment:
There is the unit `m` at the end so you don't need to write `north_m` as well.

Reply:
resolved